src/RoverPhoto/RoverTopPhoto.tsx

```diff
@@ -0,0 +1,20 @@
+
+import React from 'react';
+import { useState, useEffect } from "react";
+import './RoverPhoto.scss';
+
+function RoverTopPhoto(props) {
```

Comment:
Could you add an interface to props here to show what form the props are expected to be in? feel free to grab me if this doesn't make sense. Something like `props: RoverPhoto` and `interface RoverPhoto { // structure of RoverPhoto here }`